.travis.yml

```diff
@@ -1,14 +1,20 @@
-sudo: false
+sudo: true
 
 language: haskell
 
+services:
+  - postgresql
+
 ghc: "8.2.2"
 
 cache:
   directories:
-    - $HOME/.cabal/packages
-    - $HOME/.cabal/store
+    - $HOME/.cabal/
 
 install:
-  - cabal install happy
-  - cabal install --only-dependencies --enable-tests --allow-newer
\ No newline at end of file
+  - psql -c 'create database snap;' -U postgres
+  - psql -c 'CREATE TABLE snap_user (id SERIAL PRIMARY KEY, name text not null,email text not null,birth_date date not null,password text not null,genre text not null,income numeric not null);' -U postgres
```

Comment:
Should we run `db-migrations` here? @kinorius123

Reply:
@CristhianMotoche tried to install db-migrations into travis, but it built with an error as you can see here [TRAVIS] (https://travis-ci.com/kinorius123/snap/builds/79874685), so I just tried a different approach.

Reply:
@kinorius123 maybe we can pair a little bit if you are stuck or blocked on something... remember to always raise an issue when that happens